code/modules/xenomorph/xeno_structures.dm

```diff
@@ -81,20 +93,17 @@
 	if(associated_hive)
 		UnregisterSignal(associated_hive, list(COMSIG_HIVE_XENO_MOTHER_PRE_CHECK, COMSIG_HIVE_XENO_MOTHER_CHECK))
 
-		if(isdistressgamemode(SSticker.mode)) //Silo only matter on distress
-			var/datum/game_mode/infestation/distress/distress_mode
-			distress_mode = SSticker.mode
-			if (!(distress_mode.round_stage == INFESTATION_MARINE_CRASHING))//No need to notify the xenos shipside
-				associated_hive.xeno_message("A resin silo has been destroyed at [AREACOORD_NO_Z(src)]!", "xenoannounce", 5, FALSE,src.loc, 'sound/voice/alien_help2.ogg',FALSE , null, /obj/screen/arrow/silo_damaged_arrow)
-				associated_hive.handle_silo_death_timer()
-				associated_hive = null
-				notify_ghosts("\ A resin silo has been destroyed at [AREACOORD_NO_Z(src)]!", source = get_turf(src), action = NOTIFY_JUMP)
+	if (!(silent))//No need to notify the xenos when hijacking
```

Comment:
It means xeno will have an alert when silos are killed on hunt, i can fix that if you want